java/pom.xml

```diff
@@ -77,6 +77,10 @@
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
We should exclude it in bazel, see https://github.com/ray-project/ray/blob/master/java/dependencies.bzl#L31 as an example.